### ISSUE TITLE
Lab-02 and Unit Overview

### ISSUE THREAD

**Issue by TVK36692** [DS4PS/cpp-524-spr-2020#5]
Hi am showing my Lab-02 and Unit 2 Overview are grey and unclickable. Is there anything I am doing wrong?

**Comment by lecy** [comment]
Nope, the content is just late in getting posted. We have had some tech issues so I have had to cover another class unexpectedly. I have not been able to get things posted fast enough - apologies! 

I have moved the lab-02 deadline to next week and will send out an email when the lab is ready.

**Comment by Jigarci3** [comment]
@lecy Hi- just wanted to let you know that the link for lecture materials under week 2's unit overview- "intro to counterfactuals" does not  seem to be working.

**Comment by lecy** [comment]
try again please!

**Comment by Jigarci3** [comment]
@lecy still getting a "404- file not found" message

**Comment by lecy** [comment]
Please refresh your browser and try again.

**Comment by jmacost5** [comment]
I have a question. Which of the articles are we reviewing? Am I misunderstanding what we are suppose to be doing.
I have a question. For the first one I read, chapter 7, What would my n value be for the Bonferroni correction?

**Comment by lecy** [comment]
@jmacost5 You will review all three articles for Lab-02. Apply the same set of codes to each (the ten items on the Campbell Score). 

For CH7, they don't have a table reporting contrasts but they do report them in the text:

> The characteristics of the experimental group were as follows: females (45%); males
(55%); age (X = 29.4; SD = 10.8); psychoses (51%); neuroses (49%); global impairment
rating (X = 3.6; SD = 1.2). Similarly, the characteristics of the control group were as follows:
females (43%); males (57%); age (X = 32.8; SD = 12.1); psychoses (51%); neuroses (49%);
global impairment rating (X = 3.6; SD = 1.3).

So the contrasts are: 

1. gender (male/female
2. age 
3. diagnosis (psychoses/neuroses) 
4. impairment 

If they had reported a table, the alpha used to reject the null would be *a*=0.05 / 4 = 0.0125. The do NOT, however, report t-scores or p-values in the paper for these contrasts. 

Because of the lack of contrast stats this paper falls into the gray area in terms of meeting the Campbell Scores criteria. I would give them the benefit of the doubt since they were particular about defining their sample and applied randomization in a systematic way. None of the reported means (or proportions) above seem that far off. So I would consider the groups to be equivalent for Item 1. 

Does that all make sense and help you get started?

**Comment by castower** [comment]
Hello all, I'm reviewing the lecture notes and on page 41 of the Tests for CF Validity Lecture Slides, I'm a bit confused why n is 6. It is my understanding that n represents the number of categories, so shouldn't it be 7? Or am I missing something?

Thanks!
Courtney

**Comment by lecy** [comment]
@castower You are correct, it should be 7, not 6. Thanks for catching that! Updating now.

**Comment by castower** [comment]
@lecy thanks!

**Comment by TVK36692** [comment]
can we get solutions to this lab? i had a really hard time and want to be ready for the next assignments.

**Comment by lecy** [comment]
Absolutely. Will post as soon as assignments are in.
